package.json

```diff
@@ -95,7 +95,8 @@
     "form-data": "^4.0.4",
     "**/avalanche/**/ws": "8.18.3",
     "**/ethers/**/ws": "7.5.10",
-    "**/swarm-js/**/ws": "5.2.4"
+    "**/swarm-js/**/ws": "5.2.4",
+    "@grpc/grpc-js": "^1.12.6"
```

Comment:
if you run `yarn audit` before this change, do you see any hits for this? 

Is this actually necessary?